core/watcher/eventqueue/keys.go

```diff
@@ -59,7 +61,7 @@ func (w *KeysWatcher) Changes() watcher.StringsChannel {
 }
 
 func (w *KeysWatcher) loop() error {
-	subscription, err := w.eventQueue.Subscribe(changestream.Namespace(w.tableName, changestream.All))
+	subscription, err := w.eventQueue.Subscribe(changestream.Namespace(w.tableName, w.changeType))
```

Comment:
I wonder what happens if 0 is passed to the namespace. Guess that should be an error.

@manadart @nvinuesa maybe we should panic if you attempt to pass 0 in the changestream Namespace and just validate in the watchers that have a changetype that 0 isn't a valid change type  OR 0 implies ALL, which is probably a better outcome. Why isn't anything working vs why am I getting everything?